Metrics-based SLOs (GA 8/18) (#2725)
SumoLogic/sumologic-documentation: docs/observability/reliability-management-slo/create-slo.md

<img src={useBaseUrl('img/observability/ListOfSavedFilters.png')} alt="ListOfSavedFilters.png" />

> You can make further modifications to a saved filter view later using kebab menu options next to the funnel icon.

Author: ```suggestion
You can make further modifications to a saved filter view later using kebab menu options next to the funnel icon. 
```

> You can make further modifications to a saved filter view later using kebab menu options next to the funnel icon.

Author: ```suggestion
You can make further modifications to a saved filter view later using kebab menu options next to the funnel icon. 
```

## Create an SLO from Metrics page

To create an SLO from the **Metrics** page:

1. Click xyz...

You can use [metrics operators](/docs/metrics/metrics-operators) for metrics-based SLOs. The metrics query specified in your SLO should be a pure selector query. You can specify one or more operators in the query for SLO.

As an example, a pure selector query with no operators is `_sourceHost=stag-monitor-manager metric=CPU_Total`. That same query using the `sum` operator would be `_sourceHost=stag-monitor-manager metric=CPU_Total | sum`.

> 

Reviewer: ```That same query using the `sum` operator would be `_sourceHost=stag-monitor-manager metric=CPU_Total | sum`.```

To use operator we have a constraint that the user should provide a quantization after the selector. Example: `_sourceHost=stag-monitor-manager metric=CPU_Total | quantize to 1m using avg | sum`

Author: https://github.com/SumoLogic/sumologic-documentation/pull/2725#discussion_r1289237320

## Create an SLO from Metrics page

To create an SLO from the **Metrics** page:

> 1. Click xyz...

Author: You can create SLOs with operators, provided you use an appropriate quantization.

```suggestion
1. Go to **Manage Data** > **Metrics**.
1. Under **Metrics Explorer**, select your desired **Metric** and **Filters**. Optionally, you can **Add Operator**.
1. Click the three-dot kebab icon, then select **Create SLO**.
1. Follow the instructions under [Create an SLO (General)](#create-an-slo-general).
```

## Create an SLO from Metrics page

To create an SLO from the **Metrics** page:

> 1. Go to **Manage Data** > **Metrics**.

Reviewer 2: this step is incorrect and can be removed

> 1. Go to **Manage Data** > **Metrics**.

Reviewer 2: ```suggestion
1. Open a new or go to an existing **Metrics** tab.
```

Author: @apoorv-garg is this what you meant? 

_Click **+ New** > **Metrics** or go to an existing **Metrics** tab._

Reviewer 2: Yes

> 1. Under **Metrics Explorer**, select your desired **Metric** and **Filters**. Optionally, you can **Add Operator**.<br/><img src={useBaseUrl('img/observability/metrics-slo.png')} alt="metrics-slo.png" />

Reviewer 3: In this image, "Filters" params are not fully visible. Can we use advanced mode? That will make it easier to understand.

Author: Does this new I've uploaded suffice?

> 1. Click the three-dot kebab icon, then select **Create SLO**.

Reviewer 2: ```suggestion
1. Click the three-dot kebab icon, then select **Create an SLO**.
```

1. Follow the instructions under [Create an SLO (General)](#create-an-slo-general).

> You can use [metrics operators](/docs/metrics/metrics-operators) for metrics-based SLOs. The metrics query specified in your SLO should be a pure selector query. You can specify one or more operators in the query for SLO.

Reviewer: ```suggestion
You can use [metrics operators](/docs/metrics/metrics-operators) for metrics-based SLOs. The metrics query specified in your SLO should be have a quantization after the selector. You can specify one or more operators in the query for SLO.
```

> As an example, a pure selector query with no operators is `_sourceHost=stag-monitor-manager metric=CPU_Total`. That same query using the `sum` operator would be `_sourceHost=stag-monitor-manager metric=CPU_Total | sum`.

Reviewer: ```suggestion
As an example, a pure selector query with no operators is `_sourceHost=stag-monitor-manager metric=CPU_Total`. That same query using the `sum` operator with appropriate quantization would be `_sourceHost=stag-monitor-manager metric=CPU_Total | quantize to 1m using avg | sum`.
```

> As an example, a pure selector query with no operators is `_sourceHost=stag-monitor-manager metric=CPU_Total`. That same query using the `sum` operator with appropriate quantization would be `_sourceHost=stag-monitor-manager metric=CPU_Total | quantize to 1m using avg | sum`.

Reviewer 2: ```suggestion
As an example, a pure selector query with no operators could be `_sourceCategory=my-web-server metric=is_healthy` which returns a time series per instance your web server indicating if it is healthy or not (`1` or `0`). To count the number of instances that were healthy in a given minute, the `sum` operator can be used with an appropriate quantization method and interval as follows `_sourceCategory=my-web-server metric=is_healthy | quantize to 1m using max | sum`.
